site/profile/manifests/ccs/home.pp

```diff
@@ -4,7 +4,13 @@
   ## rather than each having their own?
   file_line { 'Change default home permissions':
     path  => '/etc/login.defs',
-    match => '^UMASK ',
+    match => '^UMASK\s',
     line  => 'UMASK 022',
   }
+  file_line { 'Change default home permissions for EL8+':
+    path               => '/etc/login.defs',
+    match              => '^HOME_MODE\s',
+    line               => 'HOME_MODE 0755',
```

Comment:
I think this would probably apply to home dirs created for IPA users as well.  What is trying to be accomplished?

Reply:
Having it apply to IPA users is the wanted behaviour.
The intent is to make the home dirs of normal users world-readable by default, since this is a collaborative enterprise and people need to share work.

Reply:
This needs to wait until Cristian returns from leave next week for further discussion.